### PR TITLE
docs: added disclaimer and table for Kubernetes support and cluster size

### LINKED ISSUE #1612: docs: update support page to include known limitations
https://github.com/keptn/keptn/issues/9525 
This is the parent issue for this but the solution exists in this repo!! In order to create a Pull request, I created this issue here !! 
cc: @thisthat @RealAnna

### PR DESCRIPTION
**Pull request by bhavyastar**
Closes #1612 
This is the parent issue for this Pull request https://github.com/keptn/keptn/issues/9525 which is an official issue on the Keptn repo but the existing solution exists in this docs repo!! 
cc: @thisthat @RealAnna

### CODE REVIEW

File: content/docs/1.0.x/install/k8s-support/index.md
Comment (StackScribe) [inline content/docs/1.0.x/install/k8s-support/index.md:23]:
```suggestion
| **1.y.z** / <br>Control & Execution plane<br>*see: (1)*   | 1.22 - 1.17 | 1.22 - 1.19 | 1.22 - 1.19 | 1.22 - 1.19   | 4, 3.11     | 1.22 - 1.19 | 1.34.2<br>(K8s: 1.11)   |
```

File: content/docs/1.0.x/install/k8s-support/index.md
Comment (StackScribe) [inline content/docs/1.0.x/install/k8s-support/index.md:24]:
```suggestion
| **1.y.z** / <br>Control plane                             | 1.22 - 1.17 | 1.22 - 1.19 | 1.22 - 1.19 | 1.22 - 1.19   | 4, 3.11     | 1.22 - 1.19 | 1.34.2<br>(K8s: 1.11)   |
```

**Review by StackScribe — COMMENTED** [review]
Looks good!  Just a couple minor comments.  Because the semantic versioning is expressed as x.y.z, it makes more sense for us to do 1.y.z.  This is a fairly recent development and the standard is not adhered to throughout the doc set but let's do this piece correctly.

Make those changes and I'll approve it.  And get Giovanni to do the "workflow approval" that is necessary for your first contribution.

**Review by thisthat — APPROVED** [review]
LGTM after the changes suggested by @StackScribe have been addressed :)

**Comment by bhavyastar** [comment]
Changes done! @StackScribe @thisthat 
Can you review this again?

**Review by StackScribe — APPROVED** [review]

**Pull request merged**
